src/silverlabnwb/nwb_file.py

```diff
@@ -488,8 +491,8 @@ def determine_trial_times(self):
             trial = 'trial_{:04d}'.format(i + 1)
             self.nwb_file.add_epoch(
                 name=trial,
-                start_time=start_time if i == 0 else np.nextafter(start_time, stop_time),
-                stop_time=np.nextafter(stop_time, stop_time * 2),
+                start_time=start_time if i == 0 else start_time + 1e-9,
+                stop_time=np.nextafter(stop_time, stop_time*2),
```

Comment:
I doubt it makes much difference, but for consistency should we provide `stop_time=stop_time + 1e-9` too?